core-blocks/video/edit.js

```diff
@@ -86,8 +101,37 @@ class VideoEdit extends Component {
 						/>
 					</Toolbar>
 				</BlockControls>
+				<InspectorControls>
+					<PanelBody title={ __( 'Playback Controls' ) }>
+						<ToggleControl
+							label={ __( 'Autoplay' ) }
+							onChange={ this.toggleAttribute( 'autoplay' ) }
+							checked={ autoplay }
+						/>
+						<ToggleControl
+							label={ __( 'Controls' ) }
+							onChange={ this.toggleAttribute( 'controls' ) }
+							checked={ controls }
+						/>
+						<ToggleControl
+							label={ __( 'Loop' ) }
+							onChange={ this.toggleAttribute( 'loop' ) }
+							checked={ loop }
+						/>
+						<ToggleControl
+							label={ __( 'Muted' ) }
+							onChange={ this.toggleAttribute( 'muted' ) }
+							checked={ muted }
+						/>
+					</PanelBody>
+				</InspectorControls>
 				<figure className={ className }>
-					<video controls src={ src } />
+					<video
+						controls={ controls }
```

Comment:
In edit mode, these attributes shouldn't be linked to the output. Check out the `Audio` edit for inspiration: https://github.com/WordPress/gutenberg/blob/master/core-blocks/audio/edit.js#L119

For instance: if you enable `loop` for the video and then click "play" in the editor the video will play constantly until you click it again. That's an annoying behaviour.

The `InspectorControls` tell you what settings are enabled, this is a good example of "what you see [should not be/is not] what you get" 😄

I think this should just stay as `<video controls src={ src } />`

Reply:
I removed `Loop`, Controls and Muted should be okay.